libnavui-androidauto/build.gradle

```diff
@@ -0,0 +1,81 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-kapt'
+apply plugin: 'kotlin-parcelize'
+apply plugin: 'org.jetbrains.dokka'
+apply plugin: 'com.jaredsburrows.license'
+apply plugin: 'com.mapbox.android.sdk.versions'
+apply from: "${rootDir}/gradle/ktlint.gradle"
+apply from: file("${rootDir}/gradle/artifact-settings.gradle")
+apply from: "${rootDir}/gradle/kdoc-settings.gradle"
+
+version = project.ext.versionName
+group = project.ext.mapboxArtifactGroupId
+
+android {
+    compileSdkVersion androidVersions.compileSdkVersion
+
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+
+    defaultConfig {
+        minSdkVersion androidVersions.carMinSdkVersion
+        targetSdkVersion androidVersions.targetSdkVersion
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles 'proguard-rules.pro', "${rootDir}/proguard/proguard-project.pro"
+    }
+
+    testOptions {
+        unitTests.returnDefaultValues = true
+        unitTests.includeAndroidResources = true
+    }
+}
+
+dependencies {
+    api(dependenciesList.mapboxMapsAndroidAuto)
+    api(project(":libnavui-base"))
+    api(project(":libnavui-maneuver"))
+    api(project(":libnavui-maps"))
+    api(project(":libnavui-speedlimit"))
+    api(project(":libnavui-tripprogress"))
+    api(project(":libnavui-voice"))
+    implementation(project(":libnavui-resources"))
+    implementation(project(":libnavui-util"))
+
+    api("com.mapbox.search:mapbox-search-android:1.0.0-beta.27")
+    api("com.mapbox.search:mapbox-search-android-ui:1.0.0-beta.27")
```

Comment:
Do we really want to depend on Search UI SDK? Looks like we only need it for some feedback drawable resources. I think we should simply copy them to Android Auto, because feedback views are likely going to be removed from Search UI SDK soon and these resources are internal anyway (IDE shows a warning).

Reply:
Yeah I agree. I don't want to change it here though, either we change it in 1tap and copy-paste to update here. Or we merge as is and consider it follow up.

Transitioning the work from 1tap to nav-sdk

Reply:
Ok went ahead with it, updated!